HPXMLtoOpenStudio/tests/test_airflow.rb

```diff
@@ -24,23 +24,42 @@ def get_ems_values(ems_objects, name)
         lhs, rhs = line.split('=')
         lhs = lhs.gsub('Set', '').gsub('set', '').strip
         rhs = rhs.gsub(',', '').gsub(';', '').strip
-        values[lhs] = rhs
+        values[lhs] = [] if values[lhs].nil?
+        # eg. "Q = Q + 1.5"
+        if rhs.include? '+'
+          rhs_els = rhs.split('+')
+          rhs = rhs_els.map { |s| s.to_f }.inject(0, :+)
+        else
+          rhs = rhs.to_f
+        end
+        values[lhs] << rhs
       end
     end
     assert_operator(values.size, :>, 0)
     return values
   end
 
+  def get_eed_for_ventilation(model, ee_name)
+    eeds = []
+    model.getElectricEquipmentDefinitions.each do |eed|
+      next if eed.name.to_s.include? 'cfis'
```

Comment:
This line handles the situation where cfis object is returned when we actually want the combined electric equipment object for all other whole house ventilation fans. This is because CFIS fan constant name is `mech vent house fan cfis`, while other whole house ventilation fan name is `mech vent house fan`, which is a substring of cfis name. 🤦 
I've tried out other approaches, eg. using equal("==") instead of "include?" but the bath/kitchen fans can be multiple, which ended up with `mech vent bath fan 1` etc. Therefore, in order to implement the minimum change, I ended up with two lines here.

Reply:
Ah, I understand. Works for me.